packages/core/getBusyTimes.ts

```diff
@@ -124,6 +125,18 @@ export async function getBusyTimes(params: {
                 },
               },
             },
+            {
+              startTime: { lte: endTimeDate },
+              endTime: { gte: startTimeDate },
+              eventType: {
+                id: eventTypeId,
+                requiresConfirmation: true,
+                requiresConfirmationWillBlockSlot: true,
+              },
+              status: {
+                in: [BookingStatus.PENDING],
+              },
+            },
```

Comment:
IF the event type ID is the event type we are booking from AND requires confirmation AND block slot. Pull back pending bookings between the time range